app/renderer/js/pages/preference/shortcuts-section.ts

```diff
@@ -106,39 +114,63 @@ export default class ShortcutsSection extends BaseSection {
 								<table>
 									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>R</kbd></td>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>R</kbd></td>
 										<td>${t.__('Reload')}</td>
 									</tr>
-									<tr>
-										<td><kbd>Shift</kbd><kbd>${userOSKey}</kbd><kbd>R</kbd></td>
+									<tr ${process.platform === 'darwin' ? 'hidden' : ''}>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>Shift</kbd> + <kbd>R</kbd></td>
 										<td>${t.__('Hard Reload')}</td>
 									</tr>
-									<tr>
-										<td><kbd>Control</kbd><kbd>${userOSKey}</kbd><kbd>F</kbd></td>
+									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
+										<td><kbd>Shift</kbd> + <kbd>${cmdOrCtrl}</kbd> + <kbd>R</kbd></td>
+										<td>${t.__('Hard Reload')}</td>
+									</tr>
+									<tr ${process.platform === 'darwin' ? 'hidden' : ''}>
+										<td><kbd>F11</kbd></td>
+										<td>${t.__('Toggle Full Screen')}</td>
+									</tr>
+									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
+										<td><kbd>Control</kbd> + <kbd>${cmdOrCtrl}</kbd> + <kbd>F</kbd></td>
 										<td>${t.__('Enter Full Screen')}</td>
 									</tr>
-									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>+</kbd></td>
+									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>+</kbd></td>
+										<td>${t.__('Zoom In')}</td>
+									</tr>
+									<tr ${process.platform === 'darwin' ? 'hidden' : ''}>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>=</kbd></td>
 										<td>${t.__('Zoom In')}</td>
 									</tr>
 									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>-</kbd></td>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>-</kbd></td>
 										<td>${t.__('Zoom Out')}</td>
 									</tr>
 									<tr>
-										<td><kbd>${userOSKey}</kbd><kbd>0</kbd></td>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>0</kbd></td>
 										<td>${t.__('Actual Size')}</td>
 									</tr>
-									<tr>
-										<td><kbd>${userOSKey}</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd></td>
+									<tr ${process.platform === 'darwin' ? 'hidden' : ''}>
+										<td><kbd>${cmdOrCtrl}</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd></td>
 										<td>${t.__('Toggle Sidebar')}</td>
 									</tr>
-									<tr>
-										<td><kbd>Option</kbd><kbd>${userOSKey}</kbd><kbd>I</kbd></td>
+									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
+										<td><kbd>Shift</kbd> + <kbd>${cmdOrCtrl}</kbd> + <kbd>S</kbd></td>
+										<td>${t.__('Toggle Sidebar')}</td>
+									</tr>
+									<tr ${process.platform === 'darwin' ? '' : 'hidden'}>
+										<td><kbd>Option</kbd> + <kbd>${cmdOrCtrl}</kbd> + <kbd>I</kbd></td>
 										<td>${t.__('Toggle DevTools for Zulip App')}</td>
 									</tr>
-									<tr>
-										<td><kbd>Option</kbd><kbd>${userOSKey}</kbd><kbd>U</kbd></td>
+									<tr ${process.platform === 'darwin' ? 'hidden' : ''}>
+										<td><kbd>Shift</kbd> + <kbd>${cmdOrCtrl}</kbd> + <kbd>I</kbd></td>
```

Comment:
Should be `Ctrl`+`Shift`.

Reply:
My bad, I checked it multiple times but I don't know how I missed it.